getting_started.ipynb

```diff
@@ -0,0 +1,1962 @@
+      ]
+     },
+     "execution_count": 19,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "response = model.invoke(prompt_value)\n",
+    "response.content"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cb6794d6-ba1f-4367-bae6-af0136212713",
+   "metadata": {},
+   "source": [
+    "#### Assigning Roles Using LangChain Messages\n",
+    "\n",
+    "Compared to hardcoding the roles like above, LangChain Messages allow for more flexibility and better management, especially with complex conversations involving multiple roles. It also simplifies the visualization of the conversation flow.\n",
+    "\n",
+    "It is therefore recommended to use LangChain messages where possible.\n",
+    "\n",
+    "**Basic Message Types**\n",
+    "\n",
+    "|             |                                                                 |\n",
+    "|-------------|-----------------------------------------------------------------|\n",
+    "| `SystemMessage` | Set how the AI should behave (appropriate wording, tone, style, etc.) |\n",
+    "| `HumanMessage`  | Message sent from the user                                      |\n",
+    "| `AIMessage`     | Message from the AI chat model (context setting, guidance for response) |\n",
+    "\n",
+    "For more info, see [**Message Types**](https://python.langchain.com/v0.1/docs/modules/model_io/chat/message_types/) and [**API Reference**](https://api.python.langchain.com/en/latest/core_api_reference.html#module-langchain_core.messages)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "95152680-ebc2-4106-99bf-679547d0d1fe",
+   "metadata": {},
+   "source": [
+    "#### `base message` and `MessagePromptTemplate`\n",
+    "We can also pass a `base message` or `MessagePromptTemplate` instead of tuples."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 47,
+   "id": "b956d584-881a-4e9b-ae81-565ffdd7bf4a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain_core.prompts import HumanMessagePromptTemplate\n",
+    "\n",
+    "chat_template = ChatPromptTemplate.from_messages(\n",
+    "    [\n",
+    "        SystemMessage(\n",
+    "            content=(\"You are a translator. You are to translate the text into English.\"\n",
+    "            )\n",
+    "        ),\n",
+    "        HumanMessagePromptTemplate.from_template(\"{text}\"),\n",
+    "    ]\n",
+    ")\n",
+    "\n",
+    "prompt_value = chat_template.format_messages(text=\"アサヒスーパードライは日本のビールのです。\")\n",
```

Comment:
Here's another case where it might be best to avoid calling out a specific brand or alcohol. Maybe something a bit more generic like "Yuzu is a Japanese fruit" or "baseball is a popular sport in Japan"

Reply:
"Yuzu is a popular Japanese fruit" is the new line :)